spec/controllers/results_controller_spec.rb

```diff
@@ -553,6 +737,46 @@ def self.test_unauthorized(route_name)
         expect(data['assigned_criteria']).to eq ["#{rubric_criterion.class}-#{rubric_criterion.id}"]
       end
 
+      context 'when accessing an assignment with deductive annotations' do
+        let(:assignment) { create(:assignment_with_deductive_annotations) }
+        it 'receives limited annotation category data if has ta_criterion_association' do
```

Comment:
Change 'if has ta_criterion_association' to 'when assigned to a subset of criteria that have associated categories'